synapse/handlers/room.py

```diff
@@ -908,34 +903,34 @@ def get_event_context(self, user, room_id, event_id, limit, event_filter):
         before_limit = math.floor(limit / 2.0)
         after_limit = limit - before_limit
 
-        users = yield self.store.get_users_in_room(room_id)
+        users = await self.store.get_users_in_room(room_id)
         is_peeking = user.to_string() not in users
 
         def filter_evts(events):
             return filter_events_for_client(
                 self.storage, user.to_string(), events, is_peeking=is_peeking
             )
 
-        event = yield self.store.get_event(
+        event = await self.store.get_event(
             event_id, get_prev_content=True, allow_none=True
         )
         if not event:
             return None
 
-        filtered = yield (filter_evts([event]))
+        filtered = await (filter_evts([event]))
```

Comment:
Are the parentheses around `filter_evts([event])` necessary here?

Reply:
Hm...they likely aren't, I kind of just assumed black would take care of it though. I'll poke at it.

Reply:
Yeah, didn't complain! 👍